.travis.yml

```diff
@@ -53,12 +53,6 @@ before_script:
 
 # Do the tests
 script:
-  # Formatting
-  - pep8 wger
-
-  # Javascript linting
-  - gulp lint
-
```

Comment:
Sory we forgot to return it, we managed to ignore the minified js library for the slider which was causing errors on gulp lint.